src/popper/commands/cmd_translate.py

```diff
@@ -0,0 +1,51 @@
+import click
+import os
+
+from popper.cli import pass_context, log
+from popper.parser import WorkflowParser
+from popper.translators.translater import WorkflowTranslator
+
+
+@click.command(
+    "translate",
+    short_help="Translate workflow configuration files of different runners.",
+)
+@click.option(
+    "infmt",
+    "--from",
+    help="Format of the input file.",
+    type=click.Choice(["popper"]),
+    default="popper",
+)
+@click.option(
+    "infile",
+    "--file",
+    "-f",
+    help="File containing the definition of the workflow.",
+    required=False,
+    default=".popper.yml",
+)
+@click.option(
+    "outfmt",
+    "--to",
+    help="Format of the output file.",
+    type=click.Choice(["drone"]),
+    default="drone",
+)
+@click.option(
+    "outfile",
```

Comment:
Maybe it is a good idea to keep the names of the arguments and the flags the same for the sake of consistency in the codebase.

Reply:
@JayjeetAtGithub 

I renamed all options because `from` is a reserved word in Python. I personally think `translate --from --to` is natural, but do you think it's better to

- rename the `from` option to something else and use the same name for the argument and flag

or

- only rename the argument for `from`